Add `Monthly Beatmapping Contest: May 2025` news post (#13249)
ppy/osu-wiki: news/2025/2025-05-12-monthly-beatmapping-contest-may-2025.md

This article covers results from the last contest, plus criteria for this contest. Skip around with these links:

- [Previous osu! contest results](#osu!-results) (see [announcement news article](https://osu.ppy.sh/home/news/2025-02-17-monthly-beatmapping-contest-february-2025#osu!))
- [Previous osu!taiko contest results](#osu!taiko-results) (see [announcement news article](https://osu.ppy.sh/home/news/2025-01-23-monthly-beatmapping-contest-january-2025#osu!taiko))
- [**Criteria for this month's MBC**](#game-modes)

## Previous contest results

The **top 5 maps** from each mode are showcased in osu!(lazer)'s [Featured Artist playlists](/wiki/People/Featured_Artists/Featured_Artist_playlists)! Following the criteria explained on [the wiki page](/wiki/People/Featured_Artists/Featured_Artist_playlists#cumulative-leaderboard), you have the potential to earn these profile badges:

![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplosu.png) ![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/fapltaiko.png)

Submit your plays any time within the next **2 weeks**!

### osu! results

> This contest's restriction: **all sliders must be straight lines**. Due to higher-than-usual submission count, this contest started with [screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) before moving to the [judging phase](/wiki/Contests/Monthly_Beatmapping_Contest#judging).

Reviewer: ```suggestion
This contest's restriction: **all sliders must be straight lines**. Due to a higher-than-usual submission count, this contest started with [screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) before moving to the [judging phase](/wiki/Contests/Monthly_Beatmapping_Contest#judging).
```

> This contest's restriction: **all sliders must be straight lines**. Due to higher-than-usual submission count, this contest started with [screening](https://osu.ppy.sh/wiki/en/Contests/Monthly_Beatmapping_Contest#screening) before moving to the [judging phase](https://osu.ppy.sh/wiki/en/Contests/Monthly_Beatmapping_Contest#judging).

Reviewer: ```suggestion
This contest's restriction: **all sliders must be straight lines**. Due to a higher-than-usual submission count, this contest started with [screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) before moving to the [judging phase](/wiki/Contests/Monthly_Beatmapping_Contest#judging).
```

- **[Screening](/wiki/Contests/Monthly_Beatmapping_Contest#screening) (1.5 weeks):** Entries are evaluated by experienced beatmappers. If there aren't many entries, this will be skipped in favour of a longer judging period.
- **[Judging](/wiki/Contests/Monthly_Beatmapping_Contest#judging) (1.5 weeks):** Top entries from screening are judged in detail by a different crew of experienced beatmappers.
- **[Rewards](#rewards) (~2 months from now):** Winners are announced at the start of another MBC.

## Beatmapping

### Limitation

Limitation, restriction, gimmick, concept — what you call it doesn't matter. This is a contest-specific mapping rule that all submissions must abide by, ensuring that this month's contest is different from last month's, and the one before that, and the one before that, and... *(message truncated)*

If you're confused about how these work, ask for clarification in the comments, or send a message to [pishifat](https://osu.ppy.sh/community/chat?sendto=3178418) (for osu!) or [Hivie](https://osu.ppy.sh/community/chat?sendto=14102976) (for osu!taiko).

#### osu!

> According to the [ancient mapping lore](/wiki/Beatmapping/Mapping_techniques/Basics#jumps), mappers in ye olde days would sometimes use *new combos* to indicate *jumps*. In the current mapping climate, jumps are used everywhere, so that concept has been thrown out the window... but the MBC is here to bring it back!

Reviewer: yes

```suggestion
According to the [ancient mapping lore](/wiki/Beatmapping/Mapping_techniques/Basics#jumps), mappers in ye olden days would sometimes use *new combos* to indicate *jumps*. In the current mapping climate, jumps are used everywhere, so that concept has been thrown out the window... but the MBC is here to bring it back!
```
